support/Pipelines/Bbh/FindHorizon.py

```diff
@@ -155,6 +162,38 @@ def find_horizon(
             break
         else:
             raise RuntimeError(f"Horizon finder failed with status {status}.")
+    # Compute horizon quantities
+    # This is independent of the horizon find and could move into a separate
+    # function, or disabled on request, if we ever need to find a horizon
+    # without computing these quantities.
+    (
+        spatial_metric,
+        inv_spatial_metric,
+        extrinsic_curvature,
+        spatial_christoffel_second_kind,
+        spatial_ricci,
+    ) = interpolate_tensors_to_points(
+        h5_files,
+        subfile_name,
+        observation_id=obs_id,
+        target_points=cartesian_coords(strahlkorper),
+        tensor_names=tensor_names,
+        tensor_types=[
+            tnsr.ii[DataVector, 3],
+            tnsr.II[DataVector, 3],
+            tnsr.ii[DataVector, 3],
+            tnsr.Ijj[DataVector, 3],
+            tnsr.ii[DataVector, 3],
+        ],
+    )
+    quantities = horizon_quantities(
+        strahlkorper,
+        spatial_metric=spatial_metric,
+        inv_spatial_metric=inv_spatial_metric,
+        extrinsic_curvature=extrinsic_curvature,
+        spatial_christoffel_second_kind=spatial_christoffel_second_kind,
+        spatial_ricci=spatial_ricci,
+    )
     # Write the horizon to a file and return it
     if output:
         assert output_coeffs_subfile or output_coords_subfile, (
```

Comment:
Can you add an `.h5` extension to the output if there isn't one already? Ran into this today.